docs_src/OVMS/capiYolov5EnsemblePipelineRun.md

```diff
@@ -0,0 +1,61 @@
+# OpenVINO OVMS C-API Yolov5 Ensemble Pipeline Run
+
+OpenVINO Model Server has [many ways to run inferencing pipeline](https://docs.openvino.ai/2023.1/ovms_docs_server_api.html):
+TensorFlow Serving gRPC API, KServe gRPC API, TensorFlow Serving REST API, KServe REST API and OVMS C API through OpenVINO model server (OVMS). Here we are demonstrating for using OVMS C API method to run inferencing pipeline yolov5s ensemble model in following steps:
+
+1. Add new section to model configuration file for model server
+2. Add pipeline specific files
+3. Add environment variable file dependency
+4. Add a profile launcher pipeline configuration file
+5. Build and run
+
+
+## Add New Section To Model Config File for Model Server
+
+The model template configuration file has been updated with model configs of yolov5, efficientnetb0_FP32INT8 and custom configurations, please view [`configs/opencv-ovms/models/2022/config_template.json`](https://github.com/intel-retail/automated-self-checkout/blob/main/configs/opencv-ovms/models/2022/config_template.json) for detail.
+!!! Note
+    New model yolov5 is similar to yolov5s configuration except the layout difference.
```

Comment:
could we make the layout the same?

Reply:
I tried, but not successful, i think it is binding with efficientnet b0 model as well.

Reply:
this will create some confusing if we have multiple slightly different versions in the same json file just because of layout IMO

Reply:
we can work on it together to optimize it after this basic one merged

Reply:
created spike issue https://github.com/intel-retail/automated-self-checkout/issues/380